src/auth/dtos/userDto.ts

```diff
@@ -17,6 +24,6 @@ export interface UserResponseDTO {
   isAdmin: boolean;
   company: {
     companyName: string;
-    companyCode: string;
+    code: string;
```

Comment:
```suggestion
    companyCode: string;
```
프론트에서 companyCode로 요청이 오는거 같은데 혹시 바꾸신 이유가 있으신가요?